Formula/hg-fast-export.rb

```diff
@@ -0,0 +1,19 @@
+class HgFastExport < Formula
+  desc "Mercurial to git converter using git-fast-import"
+  homepage "https://packages.debian.org/source/sid/hg-fast-export"
+  url "http://http.debian.net/debian/pool/main/h/hg-fast-export/hg-fast-export_20140308.orig.tar.gz"
```

Comment:
Thanks for the question @MikeMcQuaid; it’s not. Upstream has [a GitHub repo](https://github.com/frej/fast-export) with no tag 😢